db/migrate/20180607084710_nuage_subclass_l3_cloud_subnet.rb

```diff
@@ -0,0 +1,24 @@
+class NuageSubclassL3CloudSubnet < ActiveRecord::Migration[5.0]
+  class CloudSubnet < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time('Migrating Nuage default subnet type to L3 subnet subclass') do
+      CloudSubnet.where(:type => 'ManageIQ::Providers::Nuage::NetworkManager::CloudSubnet')
+                 .update_all(:type => 'ManageIQ::Providers::Nuage::NetworkManager::CloudSubnet::L3')
+    end
+  end
+
+  def down
+    say_with_time('Reverting Nuage L3 subnet subclass to default subnet type') do
+      CloudSubnet.where(:type => 'ManageIQ::Providers::Nuage::NetworkManager::CloudSubnet::L3')
+                 .update_all(:type => 'ManageIQ::Providers::Nuage::NetworkManager::CloudSubnet')
+    end
+
+    say_with_time('Reverting Nuage L2 subnet subclass to default subnet type') do
+      CloudSubnet.where(:type => 'ManageIQ::Providers::Nuage::NetworkManager::CloudSubnet::L2')
+        .update_all(:type => 'ManageIQ::Providers::Nuage::NetworkManager::CloudSubnet')
```

Comment:
@miha-plesko  looks like rubocop wants this also aligned

Reply:
Miscusi, fixed. I knew all those "Avoid using update_all" are legit but didn't notice the Align one for some reason.